src/components/Stepper.tsx

```diff
@@ -11,13 +17,21 @@ interface Props {
   theme: typeof theme;
   style?: StyleProp<ViewStyle>;
   onChange?: (value: number) => void;
+  iconSize?: number;
+  iconColor?: string;
+  borderRadius?: number;
+  typeStyle?: StyleProp<TextStyle>;
 }
 
 const Stepper: React.FC<Props> = ({
   value = 0,
   style,
   onChange,
   theme: { colors, typography, spacing },
+  iconSize = Config.stepperButtonSize,
```

Comment:
```suggestion
  iconSize = 24,
```

the Config file is kinda worthless